src/Hit/Git/Status.hs

```diff
@@ -87,14 +90,19 @@ It also handles special case of binary files. Typical raw text returned by @git@
 can look like this:
 
 @
- .foo.un~  | Bin 0 -> 523 bytes
- README.md |   4 ++++
- foo       |   1 +
+ .foo.un~   | Bin 0 -> 523 bytes
+ README.md  |   4 ++++
+ foo        |   1 +
+ bar => baz |   2 --
 @
 -}
 parseDiffStat :: [Text] -> Maybe DiffStat
 parseDiffStat = \case
     [diffStatFile, diffStatCount, diffStatSigns] -> Just DiffStat{..}
+    _:"=>":diffStatFile:diffStatCount:rest -> Just DiffStat
+        { diffStatSigns = unwords rest
```

Comment:
I believe it should be safe to write pattern-matching like this and remove a call to `unwords`:

```haskell
[_ , "=>", diffStatFile, diffStatCount, diffStatSigns]
```

Reply:
If a file is renamed with no changes, there are no +/- signs. I'll add an example to the documentation.